### DISCUSSION TITLE
PyBaMM import error - Cannot run anything

### DISCUSSION THREAD

**Discussion by abhishek-appana** [pybamm-team/PyBaMM#3928] [Q&A] [answered]
Hello,

I am running Python 3.11.8 and installed PyBaMM 24.1 using pip on a Windows system. Earlier I was using Python 3.10 and everything was fine. But I am getting the below error after upgrading Python.

When I run `import pybamm`, it gives me the error:

ModuleNotFoundError: Optional dependency pybtex.database is not available. See https://docs.pybamm.org/en/latest/source/user_guide/installation/index.html#optional-dependencies for more details.

I tried installing pybamm using `pip install pybamm[all]`. I was still getting this error. How do I resolve this?

**Comment by Saransh-cpp** [comment]
That is weird. You should not get this error on just importing `pybamm` even if you don't have `pybtex` installed. Could you post the entire logs + executed code with and without installing pybamm with `[all]`? Please also post the output of `python -m pip list`.

**Reply by abhishek-appana** [reply, marked as answer]
Thank you for the response!

I uninstalled all the Python libraries, purged the pip cache, verified that there was nothing installed, and ran `pip install pybamm [all]`.

Now the library is loading without an error. I had been doing the same thing for 3 hours and it was throwing me that error. I have no clue how it's running now!

Anyway, thank you for the help!
